src/Message/PurchaseRequest.php

```diff
@@ -70,11 +70,14 @@ private function sign($params, $api_key) {
 
 	public function sendData($data)
     {
+		// FIXME: You're not actually sending any data here.
```

Comment:
Yeah, and I'm certainly not saying that there are no gateways that do it this way, and I haven't checked the QuickPay documentation in detail so I'm not certain whether they do or don't, but I think we can all agree that "kind of crazy" is a pretty good summary. However I have seen stranger things from gateways in the past and there seems to be no such thing as an "industry standard".